src/nuklear_color.c

```diff
@@ -33,27 +33,45 @@ nk_rgba(int r, int g, int b, int a)
     return ret;
 }
 NK_API struct nk_color
-nk_rgb_hex(const char *rgb)
-{
-    struct nk_color col;
-    const char *c = rgb;
-    if (*c == '#') c++;
-    col.r = (nk_byte)nk_parse_hex(c, 2);
-    col.g = (nk_byte)nk_parse_hex(c+2, 2);
-    col.b = (nk_byte)nk_parse_hex(c+4, 2);
+nk_rgb_hex(struct nk_slice hex)
+{
+    struct nk_color col = {0};
+    NK_ASSERT(hex.ptr);
+    NK_ASSERT(hex.len);
+    if (!hex.ptr || !hex.len)
+        return col;
+    if (hex.ptr[0] == '#')
+        hex = nk_substr(hex, 1, hex.len);
```

Comment:
hex.ptr++, hex.len--?  Maybe it happens enough to justify a macro?  still probably not worth it though.

Reply:
Hmm. I'll leave these small optimizations for future PR for when people have found that the current behavior is actually inefficient.